tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2015",
```

Comment:
https://microsoft.github.io/TypeScript-New-Handbook/reference/compiler-options/#compiler-options
According to microsoft (All modern browsers support all ES6 features, so ES6 is a good choice) 
-Additional APIs available in ES2015 (also known as ES6) - array.find , Promise , Proxy , Symbol